src/prefect/events/clients.py

```diff
@@ -653,8 +654,8 @@ async def _reconnect(self) -> None:
         from prefect.events.filters import EventOccurredFilter
 
         self._filter.occurred = EventOccurredFilter(
-            since=pendulum.now("UTC").subtract(minutes=1),
-            until=pendulum.now("UTC").add(years=1),
+            since=now("UTC") - timedelta(minutes=1),
+            until=now("UTC") + timedelta(weeks=52),
```

Comment:
Ah, this is one area where `pendulum` is helping us. We might need a specific utility for adding a year for now.

Reply:
added in [5bd520e](https://github.com/PrefectHQ/prefect/pull/17103/commits/5bd520e973326c35435544ffa6fec26591880b1b)